Makefile

```diff
@@ -39,6 +39,13 @@ metrics:
 integration: ginkgo
 	./ginkgo ./integration/docker/ -- -timeout ${TIMEOUT}
 
+openshift:
+	cd integration/openshift
+	./init.sh
+	# Sleep 30 seconds to let the master and node initialize correctly
+	sleep 30
```

Comment:
This looks a bit odd and potentially fragile. Is there no way to check when the nodes have initialised fully (a command to run, particular log message(s) to look for, etc)?

Reply:
not sure, but will take a look

Reply:
changed the approach to verify if the node is up every 5 secs until a maximum of 30s.